### PR TITLE
feature/637

### LINKED ISSUE #637: New Airport: PANC (Anchorage Intl.)
Ted Stevens Anchorage International Airport (IATA: ANC, ICAO: PANC, FAA LID: ANC) is a major airport in the U.S. state of Alaska, located 5 miles (8 km) southwest of downtown Anchorage.

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #637 

### Completion Checklist
- [x] terrain file
- [x] polygonal airspace boundary
- [x] all SIDs
- [x] all STARs
- [x] information to validate traffic levels
- [x] base video map
- [x] no console errors or warnings

---

According to Wikipedia, PANC has 261,961 yearly operations. Traffic level should fall between 30acph and 90acph. Currently, there's a total of 32.5acph 👍

### CODE REVIEW

File: assets/airports/panc.json [hunk 1]
```diff
@@ -0,0 +1,2164 @@
...
+                ["MDO", "SOBGE"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "panc",
+            "destination": "",
+            "category": "departure",
+            "route": "PANC.ANC8.BGQ..FAI",
```
Comment (erikquinn) [inline assets/airports/panc.json]: `BGQ` is not an exit of the ANC8, so this is causing some errors. Perhaps we could do `PANC.ANC8.TED..BGQ..FAI`? Not like they would ever actually fly like that, but it wouldn't surprise me in the least if they were filed like that, and then just cleared direct BGQ upon contacting departure.

File: assets/airports/panc.json [hunk 2]
```diff
@@ -0,0 +1,2164 @@
...
+                ["N61.08279000", "W151.2521950"],
+                ["N61.21613278", "W151.2522053"],
+                ["N61.23535694", "W151.2195078"],
+                ["N61.61776556", "W150.5577753"]
+            ]
+        }
+    ],
+    "fixes": {
+        "_RWY25L072TED4DME": ["N61.16874", "W149.82254"],
+        "_RWY25R072TED4DME": ["N61.17059", "W149.82275"],
```
Comment (erikquinn) [inline assets/airports/panc.json]: These headings _are_ the headings of the runways, so we can probably get away with the more simple `_RWY25L04DME` etc. Your call though.

File: assets/airports/panc.json [hunk 3]
```diff
@@ -0,0 +1,2164 @@
...
+                ["NODLE*"]
+            ]
+        },
+        "NOEND3": {
+            "icao": "NOEND3",
+            "name": "No End Three",
+            "rwy": {
+                "33": []
+            },
+            "body": [["EGKAJ", "A30+|S230-"], ["LIFFE", "S230-"], ["OUTER", "A60+"], ["CRAFT", "A80+"], ["RAMMA", "A100+"]],
```
Comment (erikquinn) [inline assets/airports/panc.json]: no speed restriction at `EGKAJ`

File: assets/airports/panc.json [hunk 4]
```diff
@@ -0,0 +1,2164 @@
...
+                ["JOH", "DENSN", "PTERS"],
+                ["REMBY", "PTERS"],
+                ["TIEKL", "PTERS"],
+                ["TMSON", "PTERS"],
+                ["MDO", "DENSN"]
+            ]
+        },
+        "TAGER6": {
+            "icao": "TAGER6",
+            "name": "Tagger Six",
```
Comment (erikquinn) [inline assets/airports/panc.json]: This is now the `TAGER8`. Just a name change needed, the content is still accurate.

**Comment by Fechulo** [comment]
@erikquinn when switching between airports, the traffic of the previous airport is not removed and just freezes middair, it's not specific to this airport.

File: assets/airports/panc.json [hunk 5]
```diff
@@ -0,0 +1,2164 @@
...
+            },
+            "body": [["EGKAJ", "A30+|S230-"], ["LIFFE", "S230-"], ["OUTER", "A60+"], ["CRAFT", "A80+"], ["RAMMA", "A100+"]],
+            "exitPoints": {
+                "NOEND": [["NOEND", "A128+"]]
+            },
+            "draw": [
+                ["EGKAJ", "LIFFE", "OUTER", "CRAFT", "RAMMA", "NOEND*"]
+            ]
+        },
+        "TURN7": {
```
[the remark this answers is not in the document]
Comment (Fechulo) [reply assets/airports/panc.json]: [Here's](https://skyvector.com/files/tpp/1710/pdf/00375GAP.PDF
) the chart for the GAPP7 departure out of SFO (yep, only 4 letters)
The only place I was able to find the ICAO for this SID, said TURN7 so I'd stick with that.

File: assets/airports/panc.json (hunk 4 above)
Comment (Fechulo) [reply assets/airports/panc.json]: It wasn't renamed, I just numbered it incorrectly :man_facepalming:

File: assets/airports/panc.json (hunk 1 above)
Comment (Fechulo) [reply assets/airports/panc.json]: Yeah this was meant to be `PANC.ANC8.TED..BGQ..FAI`

File: assets/airports/panc.json (hunk 2 above)
Comment (Fechulo) [reply assets/airports/panc.json]: Done 👍 much simpler

File: assets/airports/panc.json (hunk 3 above)
Comment (Fechulo) [reply assets/airports/panc.json]: done 👍

File: assets/airports/panc.json (hunk 4 above)
Comment (Fechulo) [reply assets/airports/panc.json]: done 👍

File: assets/airports/panc.json (hunk 1 above)
Comment (Fechulo) [reply assets/airports/panc.json]: done 👍

File: assets/airports/panc.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (erikquinn) [reply assets/airports/panc.json]: where did you find `TURN7` posted?

[the remark this answers is not in the document]
Comment (Fechulo) [reply assets/airports/panc.json]: I think I checked in the airac cycle files provided by navigraph, but I'm not too sure.

File: assets/airports/panc.json [hunk 6]
```diff
@@ -0,0 +1,2164 @@
...
+                ["n/fastGA", 8],
+                ["ups", 15],
+                ["yzr", 10]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "PANC",
+            "category": "arrival",
+            "route": "TKA.TAGER6.PANC",
```
Comment (erikquinn) [inline assets/airports/panc.json]: TAGER8, silly goose 😄

Comment (Fechulo) [reply assets/airports/panc.json]: I don't even have an excuse :man_facepalming:

**Pull request merged**